standard.md

```diff
@@ -6,17 +6,17 @@
-**9. Do No Harm By Design** | All projects must demonstrate that they have taken steps to ensure the project anticipates, prevents, and does no harm.
-**9a) Data Privacy & Security** | Projects collecting data must identify the types of data collected and stored. Projects must also demonstrate how they ensure the privacy and security of this data in addition to the steps taken to prevent adverse impacts resulting from it’s collection, storage and distribution.
-**9b) Inappropriate & Illegal Content** | Projects that collect, store or distribute content must have policies identifying inappropriate and illegal content such as child sexual abuse materials in addition to mechanisms for detecting, moderating and removing inappropriate/illegal content.
-**9c) Protection from Harassment** | If the project facilitates interactions with or between users or contributors there must be a mechanism for users and contributors to protect themselves against grief, abuse, and harassment. The project must have a mechanism to address the safety and security of underage users. 
+**1. Relevance to Sustainable Development Goals** | Digital public goods must be designed and developed to advance the [Sustainable Development Goals (SDGs)](https://sdgs.un.org/goals) and demonstrate such by providing links/documentation that support their relevance. 
+**2. Use of Approved Open Licenses** | Digital public goods must demonstrate the use of an approved open license. For open-source software, only [OSI approved licenses](https://opensource.org/licenses) are accepted. For open content collections the use of a [Creative Commons license](https://creativecommons.org/licenses/) is required. DPGs are encourageed to use a license that allows for both derivatives and commercial reuse ([CC-BY](https://creativecommons.org/licenses/by/4.0/) and [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/)), or dedicate content to the public domain ([CC0](https://creativecommons.org/choose/zero/)); licenses that do not allow for commercial reuse ([CC-BY-NC](https://creativecommons.org/licenses/by-nc/4.0/) and [CC-BY-NC-SA](https://creativecommons.org/licenses/by-nc-sa/4.0/)) are also accepted. For open data, an [Open Data Commons approved license](https://opendefinition.org/licenses/) is required. See [The full license list](https://github.com/unicef/publicgoods-candidates/blob/master/docs/licenses.md) for reference.
+**3. Clear Ownership** | Ownership of assets that the digital public good produces must be clearly defined and documented. For example, through copyright, trademark or other publicly available information.
+**4. Platform Independence** | When the digital public good has mandatory dependencies that create more restrictions than the original license, proving independence from the closed component(s) and/or indicating the existence of functional, open alternatives that can be used without significant changes to the core product is required.
```

Comment:
My past understanding regarding (4) was that the "must be able to demonstrate" meant an actual physical demonstration that the DPG product would still be functional independently of the proprietary dependency/component, or with an open source-licensed alternative. As I read "proving independence" it seems less clear that this level of functionality would still be required.

Reply:
An actual physical demonstration was never part of the process, but it has always been our intent that DPGs remain vendor-neutral. We hope this new language still conveys the same imperative that vendor lock-in is not acceptable. What would you change to remove this ambiguity?